backend/app/donation/routes.py

```diff
@@ -0,0 +1,100 @@
+from flask import Flask, jsonify, request, abort, make_response
+from . import donation
+from .. import db
+from app.models import Donation
+import uuid
+from datetime import datetime
+
+# Get all donations of user_id
+@donation.route("/donations/<user_uuid>", methods=["GET"])
```

Comment:
hmm this might be referring to get all donations by user so we can sum the total amount.. However, we don't have an amount field yet (task is assigned to @SymPiracha)

Reply:
@CurtisMIT I think we will actually need this for #105, what do you think?

![image](https://user-images.githubusercontent.com/33935299/106366443-af70a400-62f0-11eb-8b38-6b1d588fb861.png)

Reply:
@celinehuang Is MuUser the model for donors?

Reply:
No it is not @CurtisMIT MuUser is just a Mu employee